.github/workflows/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   unit-tests:
     runs-on: ubuntu-latest
     container:
-      image: docker://nrel/openstudio:3.5.0
+      image: docker://nrel/openstudio:3.5.1
     steps:
       - uses: actions/checkout@v2
```

Comment:
If you update all of these to `actions/checkout@v3`, it should get rid of the CI warnings.